binding/header.go

```diff
@@ -0,0 +1,18 @@
+package binding
+
+import "net/http"
+
+type headerBinding struct{}
+
+func (headerBinding) Name() string {
+	return "header"
+}
+
+func (headerBinding) Bind(req *http.Request, obj interface{}) error {
```

Comment:
I think, is a good idea to add unit tests inside of `binding` package to cover it by 100% inside.

Reply:
ok

Reply:
Please make a coverage of `binding` package to 100%. ([binding/header.go](https://codecov.io/gh/gin-gonic/gin/pull/1957/diff?src=pr&el=tree#diff-YmluZGluZy9oZWFkZXIuZ28=))

This can help you:
```
cd binding
go test -coverprofile=cover.prof
go tool cover -html=cover.prof
```

Reply:
Very good way，have submitted changes